tfgrid3/k3s/scripts/entrypoint.sh

```diff
@@ -1,18 +1,17 @@
 echo "KUBECONFIG=/etc/rancher/k3s/k3s.yaml" >> /etc/environment
 
-if [ -z "${K3S_DATA_DIR}" ]; then
-    K3S_DATA_DIR=""
-else
-    cp -r /var/lib/rancher/k3s/* $K3S_DATA_DIR
-    K3S_DATA_DIR="--data-dir $K3S_DATA_DIR --kubelet-arg=root-dir=$K3S_DATA_DIR/kubelet"
+if [ ! -z "${K3S_DATA_DIR}" ]; then
+    echo "k3s data-dir set to: $K3S_DATA_DIR"
+    cp -r /var/lib/rancher/k3s/* $K3S_DATA_DIR && rm -rf /var/lib/rancher/k3s
+    EXTRA_ARGS="--data-dir $K3S_DATA_DIR --kubelet-arg=root-dir=$K3S_DATA_DIR/kubelet"
 fi
 
 if [ -z "${K3S_FLANNEL_IFACE}" ]; then
     K3S_FLANNEL_IFACE=eth0
 fi
 
-if [ "$K3S_URL" = "" ]; then
-    k3s server --flannel-iface $K3S_FLANNEL_IFACE --no-deploy traefik $K3S_DATA_DIR >> /var/log/k3s-service.log 2>&1
+if [ -z "${K3S_URL}" ]; then
+    k3s server --flannel-iface $K3S_FLANNEL_IFACE $EXTRA_ARGS 2>&1 | tee -a /var/log/k3s-service.log 
 else
-    k3s agent --flannel-iface $K3S_FLANNEL_IFACE $K3S_DATA_DIR >> /var/log/k3s-service.log 2>&1
+    k3s agent --flannel-iface $K3S_FLANNEL_IFACE $EXTRA_ARGS 2>&1 | tee -a /var/log/k3s-service.log
```

Comment:
is better that we stop logging to /var/log/k3s-service.log, now after your changes zinit has all logs